client/state/data-layer/wpcom/comments/index.js

```diff
@@ -69,5 +138,6 @@ export const announceFailure = ( { dispatch, getState }, { siteId, postId } ) =>
 };
 
 export default {
-	[ COMMENTS_REQUEST ]: [ dispatchRequest( fetchPostComments, addComments, announceFailure ) ]
+	[ COMMENTS_REQUEST ]: [ dispatchRequest( fetchPostComments, addComments, announceFailure ) ],
+	[ COMMENTS_WRITE ]: [ dispatchRequest( writePostComment, writePostCommentSuccess, announceFailure ) ]
```

Comment:
Are we stuck with a placeholder in comment state on failure? It looks like we only currently only remove it on success.